src/components/SearchResultsPage/SearchResultsPage.js

```diff
@@ -44,6 +48,8 @@ const SearchResultPage = (props) => {
             values: ["Army", "Navy", "Air Force", "Marine Corps"]
         },
     ]
+
+    // TODO: remove placeholder images when data comes in with images
     const imgArr = [
         'https://www.abc.net.au/news/image/12739630-3x2-940x627.jpg',
```

Comment:
We need to think about what to display when an image is not available for the record, and how to configure that.